ctapipe/reco/sklearn.py

```diff
@@ -777,13 +778,34 @@ def predict_table(self, key, table: Table) -> Dict[ReconstructionProperty, Table
         fov_lon = table["hillas_fov_lon"].quantity + disp * np.cos(psi)
         fov_lat = table["hillas_fov_lat"].quantity + disp * np.sin(psi)
 
-        # FIXME: Assume constant and parallel pointing for each run
-        self.log.warning("Assuming constant and parallel pointing for each run")
+        self.log.warning("FIXME: Assuming constant and parallel pointing for each run")
+        if np.all(table["subarray_pointing_frame"] is CoordinateFrameType.ALTAZ):
```

Comment:
this won't work. `is` cannot be applied element wise and these checks will always be False.

You need to check `np.all(table["subarray_pointing_frame"] == CoordinateFrameType.ALTAZ.value)`